src/components/AskComposer.js

```diff
@@ -88,10 +86,8 @@ class AskComposer extends Component {
   }
 
   send () {
-    console.log(this.props)
     var payload = []
-    var field
-    var fieldValue
+    var field, fieldValue
```

Comment:
Doing this in JavaScript makes `fieldValue` a global. 🎉 That's why the linter should be complaining when they're on the same line.